### PR TITLE
Docs for Paid Accounts

### PR DESCRIPTION
**Pull request by abziegler**
Updates to docs to accommodate paid accounts. We should merge this when we're ready to go live with paid accounts.

### CODE REVIEW

File: perma_web/perma/templates/docs/accounts.html [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+
+      <h3 id="becoming-org" class="body-bh">Usage Limits</h3>
+      <p class="body-text">Individual accounts let you preserve up to 10 web pages per month for free.</p>
+      <p class="body-text">To preserve more pages, you need to be affiliated with an organization that qualifies for free, unlimited service or one that has a paid subscription.</p>
+
+      <h2 class="body-ah">Organizational Accounts</h2>
+
+      <h3 id="free-service-academic" class="body-bh">Free Service for Academic Organizations</h3>
+      <p class="body-text">Free organizational accounts are available to students, faculty and academic journals if their academic library is a Perma.cc registrar. Please contact your library to find out if it is already a Perma.cc registrar. If not, your library can learn more and sign up for free using this page: <a href="{% url 'libraries' %}">https://perma.cc/libraries</a>.</p>
+      <p class="body-text">Paid accounts are currently available only for qualifying law firms. If you are interested, please contact us using this page:
```
Comment (rebeccacremona) [inline perma_web/perma/templates/docs/accounts.html]: Line 53 looks like maybe it's a left-over from a previous draft? This seems repetitive with "Paid Service for Other Organizations" below, and there's no target page after the colon.

File: perma_web/perma/templates/docs/accounts.html [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+      <p class="body-text">Paid accounts are currently available only for qualifying law firms. If you are interested, please contact us using this page: 
+
+      <h3 id="free-service-courts" class="body-bh">Free Service for Courts</h3>
+      <p class="body-text">Free organizational accounts also are available to state and federal courts. If you're affiliated with a court, you can use this page to learn more and contact us about signing up: <a href="{% url 'sign_up_courts' %}">https://perma.cc/signup/courts</a>.</p>
+
+      <h3 id="paid-service" class="body-bh">Paid Service for Other Organizations</h3>
+      <p class="body-text">Law firms, publishers, news organizations and others may obtain organizational accounts through a paid subscription. To learn more about paid subscriptions, please contact us at <a href="mailto: info@perma.cc">info@perma.cc</a>.</p>
```
Comment (rebeccacremona) [inline perma_web/perma/templates/docs/accounts.html]: How about we point them to the contact form (`<a href="/contact?subject=Question%20About%20Paid%20Subscription">contact us</a>`), or to http://perma.dev/sign-up/firms here?

File: perma_web/perma/templates/docs/accounts.html [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+      <p class="body-text">To preserve web pages with Perma, you need an account. To register for a free, individual account, please visit our <a href="{% url 'sign_up' %}">signup page</a>.</p>
+      <p class="body-text">You don't need an account to view an archived page someone else made.</p>
+
+      <p></p>
+
+      <h3 id="becoming-org" class="body-bh">Usage Limits</h3>
+      <p class="body-text">Individual accounts let you preserve up to 10 web pages per month for free.</p>
+      <p class="body-text">To preserve more pages, you need to be affiliated with an organization that qualifies for free, unlimited service or one that has a paid subscription.</p>
+
+      <h2 class="body-ah">Organizational Accounts</h2>
```
Comment (rebeccacremona) [inline perma_web/perma/templates/docs/accounts.html]: In the TOC, you like to this section using `<a href="#orgs">Organizations</a>`. So, that means this heading needs an id of "orgs". `<h2 id="orgs" class="body-ah">Organizational Accounts</h2>`

File: perma_web/perma/templates/docs/accounts.html [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+  </div>
+</div>
+
+{% include 'docs/docs_menu.html' %}
+
+<div class="container cont-reading cont-fixed" id="overview">
+  <div class="row">
+    <div class="col-sm-4 reading-toc" >
+      <ul>
+        <li class="reading-toc-chapter"><a href="#orgs">Accounts</a></li>
```
Comment (rebeccacremona) [inline perma_web/perma/templates/docs/accounts.html]: Looks like this should be `#accounts` instead.

File: perma_web/perma/templates/docs/accounts.html [hunk 5]
```diff
@@ -0,0 +1,76 @@
...
+        <li class="reading-toc-section"><a href="#free-service-courts">Free Service for Courts</a></li>
+        <li class="reading-toc-section"><a href="#paid-service">Paid Service for Other Organizations</a></li>
+        <li class="reading-toc-section"><a href="#authority">Authority and Responsibilities</a></li>
+        <li class="reading-toc-section"><a href="#adding-org-users">Adding and Removing Organization Users</a></li>
+      </ul>
+    </div>
+
+    <div class="col-sm-8 reading-body">
+      
+      <h2 class="body-ah">Accounts</h2>
```
Comment (rebeccacremona) [inline perma_web/perma/templates/docs/accounts.html]: In the TOC, you link to this section using an href. So, that means this heading needs an id for the href to target, for instance "accounts". `<h2 id="accounts" class="body-ah">Accounts</h2>`

File: perma_web/perma/templates/docs/accounts.html [hunk 6]
```diff
@@ -0,0 +1,76 @@
...
+
+{% include 'docs/docs_menu.html' %}
+
+<div class="container cont-reading cont-fixed" id="overview">
+  <div class="row">
+    <div class="col-sm-4 reading-toc" >
+      <ul>
+        <li class="reading-toc-chapter"><a href="#orgs">Accounts</a></li>
+        <li class="reading-toc-section"><a href="#who-needs-account">Who Needs an Account?</a></li>
+        <li class="reading-toc-section"><a href="#">Usage Limits</a></li>
```
Comment (rebeccacremona) [inline perma_web/perma/templates/docs/accounts.html]: Looks like this should be `#usage-limits`

File: perma_web/perma/templates/docs/accounts.html [hunk 7]
```diff
@@ -0,0 +1,76 @@
...
+    <div class="col-sm-8 reading-body">
+      
+      <h2 class="body-ah">Accounts</h2>
+      <h3 id="who-needs-account" class="body-bh">Who Needs an Account?</h3>
+      <p class="body-text">To preserve web pages with Perma, you need an account. To register for a free, individual account, please visit our <a href="{% url 'sign_up' %}">signup page</a>.</p>
+      <p class="body-text">You don't need an account to view an archived page someone else made.</p>
+
+      <p></p>
+
+      <h3 id="becoming-org" class="body-bh">Usage Limits</h3>
```
Comment (rebeccacremona) [inline perma_web/perma/templates/docs/accounts.html]: Looks like the id of this heading should be `usage-limits`

**Comment by abziegler** [comment]
Changes made @rebeccacremona - should be reflected in PR now, right?

**Comment by rebeccacremona** [comment]
Yep, your new commits are right here, and everything looks good! We can merge and deploy whenever the moment seems right.

**Pull request merged**